runtime/integration-tests/src/integrations_mock/test_common.rs

```diff
@@ -54,12 +54,21 @@ use super::{ALICE, ALICE_SESSION_KEYS};
 use sp_core::sr25519;
 use sp_runtime::{DispatchError, ModuleError};
 
-// currently, we ignore all parachain staking tests in integration tests
 mod parachain_staking_tests {
     use super::*;
+    cfg_if::cfg_if! {
+            if #[cfg(feature = "calamari")] {
```

Comment:
fmt

Reply:
Also could just add to the giga import in `mod.rs` this is fine too

Reply:
fixed, thx

Reply:
good point thx, turns out the types were actually already there.
Fixed